oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchProvider.java

```diff
@@ -246,4 +250,11 @@ public String convert(final String indexName) {
             return indexName;
         }
     }
+
+    private static class SkyWalkingTagConverter implements IndexNameConverter {
+        @Override
+        public String convert(final String indexName) {
+            return "sw" + "_" + indexName;
+        }
+    }
```

Comment:
We just need to use `sw` as the default namespace, please set it in the `#prepare` method. We don't need to add this new converter. And more importantly, this name is super confusing.